src/smt/command.cpp

```diff
@@ -1258,8 +1258,8 @@ std::string GetInstantiationsCommand::getCommandName() const throw() {
 
 /* class GetSynthSolutionCommand */
 
-GetSynthSolutionCommand::GetSynthSolutionCommand() throw() {
-}
+GetSynthSolutionCommand::GetSynthSolutionCommand() throw()
```

Comment:
Minor suggestion: Could also replace `throw()` with `noexcept`.

Reply:
Let's leave that for a follow-up for the whole file. Opened Issue #1138.

Reply:
Great, thanks!